pkgs/applications/networking/mailreaders/mmh/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchurl, ncurses }:
+stdenv.mkDerivation rec {
+  pname = "mmh";
+  version = "0.4";
+  buildInputs = [ ncurses ];
+  patches = [
+    ./patches/adjust-bash-completion-script.patch
+    ./patches/fix-spelling-errors.patch
+    ./patches/multipart-ascii.patch
+  ];
+  src = fetchurl {
+    url = "http://marmaro.de/prog/mmh/files/mmh-${version}.tar.gz";
```

Comment:
```suggestion
    url = "https://marmaro.de/prog/mmh/files/mmh-${version}.tar.gz";
```

Reply:
@doronbehar Probably not. `marmaro.de` uses self-signed certificate, Nix daemon will not be happy.

Reply:
I see, after looking at their website, perhaps:

```
  src = fetchgit {
    url = "http://git.marmaro.de/mmh";
    rev = "mmh-${version}";
    sha256 = "0000000000000000000000000000000000000000000000000000";
  };
```

IMO `fetchgit` is a better choice if it's available. Not from the sense of security, but from the sense of maintainability.

That's just a personal preference of mine :) Feel free to mark this as resolved if you think it's not that important.

Reply:
Why, I agree that given that Nix does not have provisions to check GPG signatures, git is not less secure, but more convenient.

But git repository on `marmaro.de` is slow, very slow, so let's keep using tarballs.